bin/blocks-plot

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env python 
+
+from __future__ import division, print_function
+
+import dill
+import pickle
+import fnmatch
+import logging
+import readline
+import argparse
+import numpy 
+import pandas
+
+from six import iteritems
+from six.moves import input
+from collections import OrderedDict, defaultdict
+
+from blocks import config
+from blocks.utils import change_recursion_limit
+
+
+def log_channels(log):
+    """Return all channels contained in blocks.Log object
+    """
+    channels = set()
+    for t, channel, value in log:
+        channels.add(channel)
+    return channels
+
+def log_to_dataframe(log, channels=None, epochs_only=True):
```

Comment:
Doesn't it duplicate with https://github.com/bartvm/blocks/blob/master/blocks/log.py#L211? If yes, then it would be nice to move your additional functionality there.